dotCMS/src/main/java/com/dotcms/experiments/business/ExperimentsCache.java

```diff
@@ -0,0 +1,41 @@
+package com.dotcms.experiments.business;
+
+import com.dotcms.experiments.model.Experiment;
+import com.dotmarketing.business.Cachable;
+
+import java.util.List;
+
+/**
+ * Cache for {@link Experiment}
+ *
+ * @author vico
+ */
+public interface ExperimentsCache extends Cachable {
+
+    String RUNNING_EXPERIMENTS_KEY = "RUNNING_EXPERIMENTS";
+
+    /**
+     * Add a list of {@link Experiment} into the cache.
+     *
+     * @param name name to use as key
+     * @param experiments to be stored
+     */
+    void put(String name, final List<Experiment> experiments);
```

Comment:
I don't know if we really need the name parameter here, I think the method should be something like putRunningExperiments(final List<Experiment> experiments)

Reply:
Removing key for running experiments.